README.rst

```diff
@@ -34,9 +34,15 @@ are omitted.
     permissions, with a command such as ``gpasswd --add $(id -u) docker &&
     newgrp``.
 
+``fixtures/drpm-unsigned``
+    The ``createrepo`` and ``makedeltarpm`` utilities must be available.
+
 ``fixtures/rpm``
     The ``createrepo`` and ``modifyrepo`` utilities must be available.
 
+``fixtures/rpm-unsigned``
+    The ``createrepo``, ``modifyrepo`` and ``rpm-sign`` utilities must be available.
```

Comment:
:+1:

Yeah, it's an interesting choice to make. In my opinion, it's better to write the utility name, because that tends to be more stable across different distributions and through time.